tests/test_run.py

```diff
@@ -414,3 +414,13 @@ def test_run_local_path_entry_point(pipx_temp_env, caplog, root):
     run_pipx_cli_exit(["run", empty_project_path])
 
     assert "Using discovered entry point for 'pipx run'" in caplog.text
+
+
+@mock.patch("pipx.commands.run.exec_app")
+def test_run_inspect(exec_app, tmp_path):
```

Comment:
I think it is difficult to use [execvpe_mock](https://github.com/pypa/pipx/blob/1.6.0/tests/test_run.py#L26) in this (`python -i`) case.

I will appreciate your comment about this test implementation.

Reply:
It’d be quite difficult to actually test this. I would be satisfied if this just mock out the actual process invocation and just chheck the right flags are passed to the child process.